gui/webdisplay/inc/ROOT/RWebWindowsManager.hxx

```diff
@@ -93,6 +93,8 @@ public:
 
    static bool IsMainThrd();
    static void AssignMainThrd();
+
+   static void SetLoopbackMode(bool on = true);
```

Comment:
As this is the default we should offer instead a well documented, clearly named:
```suggestion
   /// WARNING:...
   static void EnablePublicPort(int portNum);
```

Reply:
Configuration of ports are more complex. You may want to have fixed number or range of numbers.
We only let change default to off via `SetLoopbackMode(false)`. Then configuration parameters can be used again. There are approximately 10 parameters which are used to configure `THttpServer`. 
Do we want to define them all through such API?

Reply:
> Configuration of ports are more complex. You may want to have fixed number or range of numbers.

Fine so `static void EnablePublicPort(int portNumFrom, int portNumTo);`

Point is, `SetLoopbackMode(bool on = true)` is the wrong interface, we need the interface to be `DoTheTerriblyDangerousThing()` not `DoTheSafeThing()`.

Reply:
There are much more parameters in rootrc for configuring THttpServer.
See [RWebWindowsManager::CreateServer()](https://root.cern/doc/master/classROOT_1_1RWebWindowsManager.html#a1f8312c0d6e454d598a6ce151c22588e) docu.

We just introduced "secure" loopback mode. And calling `SetLoopbackMode(false)` you change it. 
I can add warning in the method.
I can rename it `EnablePublicPortServing(true)`, but all remaining configuration keep as is.